docs/setup-cluster/deploy-cluster/slurm/hpc-with-agent.rst

```diff
@@ -0,0 +1,153 @@
+.. _hpc-with-agent:
+
+####################
+ Agent on Slurm/PBS
+####################
+
+As an alternative to using the HPC Launcher, you may instead utilize the Determined agent. In this
+usage model, the system administrator creates a custom resource pool for each Determined user. You
+then start a Determined agent on one or more compute nodes of the cluster using Slurm or PBS
+commands to provide the resources for your resource pool. As work is submitted to this resource
+pool, it is distributed to the set of available agents. If your Slurm/PBS job is terminated (for
+example due to a time limit) before your Determined work is completed, your Determined work remains
+in your resource pool until additional agents are started. You may add additional resources to your
+resource pool by starting additional agents on your cluster. If your Determined work is complete
+before any time limits are hit on the Slurm/PBS job providing resources, you terminate the agent
+jobs manually using Slurm/PBS commands.
+
+The primary advantages of this model are:
+
+#. You have dedicated access to the compute resources provided by the agents you start for the
+   duration of your HPC job. This can provide more predictable throughput as it avoids contention in
+   a highly utilized cluster.
+
+#. Your Determined experiments are seen by the workload manager as a single large job, rather than
+   many smaller jobs. In some HPC environments, larger jobs are given preference in workload manager
+   scheduling.
+
+#. If you have jobs of different sizes sharing the same set of resources, you reduce the potential
+   for fragmentation where larger jobs may be delayed in running because the free resources are
+   distributed across many nodes.
+
+#. It eliminates the need for impersonation which the HPC Launcher uses sudo configuration to enable
```

Comment:
Perhaps this may be clearer?

```
It eliminates the need for user impersonation, which the HPC Launcher uses to submit jobs to the Slurm or PBS workload manager on your behalf, using a sudo configuration.
```

Reply:
ok